aws/resource_aws_elasticache_parameter_group.go

```diff
@@ -233,23 +223,23 @@ func resourceAwsElasticacheParameterGroupUpdate(d *schema.ResourceData, meta int
 						CacheParameterGroupName: aws.String(d.Get("name").(string)),
 						ParameterNameValues: []*elasticache.ParameterNameValue{
 							{
-								ParameterName:  aws.String("reserved-memory-percentage"),
+								ParameterName:  aws.String("reserved-memory-percent"),
```

Comment:
Is there a bug report issue for this?

Reply:
i'll double-check but this came about when `TestAccAWSElasticacheParameterGroup_switchReservedMemoryParameter` starting failing after the method change since the `nil` getting returned was originally driving the logic away from assigning that attribute